### PR TITLE
chore: improve bug report template

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Add checkboxes for Renovate type
- Add checkboxes for Renovate platform
- Add field for specifying the Renovate version (if selfhosted)

## Context:

I notice that not all bug reports are filled out in full, we sometimes need to ask for more information.
Maybe we can make it easier to fill in the required items by making it more obvious what's needed.

Just throwing this idea out here in a PR, as that's easier than creating a discussion or issue first. 😄 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]
Like it.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -13,6 +13,25 @@ labels: 'type:bug, status:requirements, priority-5-triage'
 
 <!-- Tell us if you're using the hosted App, or if you are self-hosted Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
```

Apply this comment if you want to fix the typo.

Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
```

Apply this suggestion if you want to remove the comment.

**Review by HonkingGoose — COMMENTED** [review]
Choose your own adventure!

**Comment by HonkingGoose** [comment]
I'll stop changing things in this PR so that you can review it. 😄 😉

**Review by rarkins — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -11,7 +11,26 @@ labels: 'type:bug, status:requirements, priority-5-triage'
...
+
+- [ ] Hosted app
+- [ ] Self hosted
+
+**Renovate platform**
+
+- [ ] Azure DevOps
+- [ ] Azure DevOps Server (self-hosted)
+- [ ] Bitbucket Cloud
+- [ ] Bitbucket Server
```
Comment (JamieMagee) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Is this the self-hosted version of Bitbucket?

Comment (viceice) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Yes, `Bitbucket Cloud=bitbucket.org`  and `Bitbucket Server=self-hosted`

Comment (HonkingGoose) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: I grabbed the list of platforms from https://docs.renovatebot.com/modules/platform/

Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
- [ ] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server (self-hosted)
```

I've opened a PR (#9867) to change the https://docs.renovatebot.com/modules/platform/ source file.
This suggestion has those exact same changes.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -11,7 +11,26 @@ labels: 'type:bug, status:requirements, priority-5-triage'
...
+<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
+
+**Renovate type**
+
+- [ ] Hosted app
+- [ ] Self hosted
+
+**Renovate platform**
+
+- [ ] Azure DevOps
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
- [ ] Azure DevOps (dev.azure.com)
```

I'll use the same part from https://github.com/renovatebot/renovate/blob/main/docs/usage/modules/platform.md here as well, so we're consistent in docs and in the bug report template.

**Review by viceice — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**